engine/src/lin_time.cpp

```diff
@@ -113,7 +97,6 @@ void setTimeCompression(float tc) {
 
 bool toggle_pause() {
     static bool paused = false;
-    VS_LOG(debug, "toggle_pause() called in lin_time.cpp");
```

Comment:
Probably good to keep the log message in unless it's being too verbose.

Reply:
I thought it is too verbose, because there are VS_LOG(...) calls in both branches of the following `if() then else`

The only thing missing from those is the "called" word, everything else is there...

WDYT ?

Reply:
mostly want to make to make sure we know this is happening and can track it back to this code. If that is covered, then it can be removed.

Reply:
yeah, L118 and L122.